src/workloads/query/plan_cache/MatchEqVaryingArray.yml

```diff
@@ -0,0 +1,120 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This test was created to demonstrate a problem with the hit rate of the SBE plan cache.
+
+  It runs a query like {$match: {a: {$eq: [1]}}} where the number varies. The Classic plan
+  cache is able to reuse the same plan even as the parameter varies, but the SBE plan cache
+  treats each one separately, resulting in much more planning.
+
+  TODO more nonselective indexed predicates to increase planning time?
+  TODO more docs to increase planning time?
+
```

Comment:
[random question, not blocking] do we track number of total storage engine reads in genny? That's a metric we would see increase substantially. If not, no worries, I'm just wondering.

Reply:
I feel like I've seen that number in the perf results on Evergreen but I don't think Genny is the part that records it.  There must be something that records server metrics, but it probably doesn't know about the separate Genny phases.